package.json

```diff
@@ -135,6 +135,8 @@
     "seedrandom": "^3.0.1",
     "semver": "^7.1.1",
     "serialize-javascript": ">=2.1.1",
+    "sinon": "9.0.2",
```

Comment:
I would like to use `testdouble` insted of sinon, but that's just my personal taste.

Reply:
I had never heard of `testdouble`, but at a quick glance i like what its trying to do. I can look into it

Reply:
Do you mind if we merge this in and then I do a follow up PR to replace sinon with testdouble?